src/app/components/order-page/order-page.component.ts

```diff
@@ -0,0 +1,36 @@
+import { Component, OnInit } from '@angular/core';
+import { MenuGroup } from 'src/app/models/menu-group';
+import { AppSettingsService } from 'src/app/services/app-settings.service';
+import { MenuService } from 'src/app/services/menu.service';
+
+@Component({
+  selector: 'app-order-page',
+  templateUrl: './order-page.component.html',
+  styleUrls: ['./order-page.component.scss']
+})
+export class OrderPageComponent implements OnInit {
+  constructor(private appSettingsService: AppSettingsService, private menuService: MenuService) { }
+
+  restaurantName: string = '';
+  groupItems: MenuGroup[] = [];
+  selectedGroupNumber: number = 0;
+  selectedGroupName: string = '';
```

Comment:
instead of `selectedGroupNumber `and `selectedGroupName`, just do `selectedGroup`. And when you need the id, you can just write `this.selectedGroup.id`, and name `this.selectedGroup.name`. We do not need two properties here.